schemas/atlas/commonCoordinateSpaceVersion.schema.tpl.json

```diff
@@ -0,0 +1,112 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpaceVersion",
+  "_extends": "/core/schemas/products/researchProductVersion.schema.tpl.json",
+  "_categories": [
+    "coordinateSpace"
+  ],
+  "required": [
+    "anatomicalAxesOrientation",   
+    "axesOrigin", 
+    "digitalIdentifier",
```

Comment:
This might be tricky, e.g. Waxholm Space does not have any digitalID. Maybe best to not make it required, even though it is a VERY strong recommendation to fill it in 😉

Reply:
okay.

Reply:
done